src/components/breadcrumb-bar/utils/get-docs-breadcrumbs.ts

```diff
@@ -148,28 +177,49 @@ interface GetDocsBreadcrumbsOpts extends GetPathBreadcrumbsOpts {
   baseName: string
   productName: string
   productPath: string
+  versions?: VersionSelectItem[]
+  indexOfVersionPathPart?: number
 }
 
 function getDocsBreadcrumbs({
-  basePath,
   baseName,
+  basePath,
+  indexOfVersionPathPart,
   navData,
   pathParts,
   productName,
   productPath,
+  version,
 }: GetDocsBreadcrumbsOpts): BreadcrumbLink[] {
+  /**
+   * Generate the arguments sent to `getDocsBreadcrumbs` based on whether or
+   * not there is a version in the current path.
+   */
+  let generatedBaseName
+  let filteredPathParts
+  if (indexOfVersionPathPart >= 0) {
+    generatedBaseName = `${baseName} ${version}`
```

Comment:
⛏️ Nitpick: not sure if this is a case of design specs being out of date, but it looks like the `version` is a distinct path item:

![CleanShot 2022-06-01 at 17 53 48@2x](https://user-images.githubusercontent.com/4624598/171508430-67596755-8982-4b34-9397-fb8c58b8a3a2.png)

IMO non-blocking since we probably need to sort out the UX of where the top-level `basePath` item should then link? I guess to the most recent version:

Reply:
Bryce raised this as well here: https://github.com/hashicorp/dev-portal/pull/510#discussion_r887222493

Reply:
No worries! 💖